resources/assets/js/components/JobBuilder/jobBuilderHelpers.ts

```diff
@@ -237,3 +246,31 @@ export const isJobBuilderComplete = (
     isCriteriaComplete(criteria, locale)
   );
 };
+
+export type JobBuilderPage =
+  | "intro"
+  | "details"
+  | "env"
+  | "impact"
+  | "tasks"
+  | "skills"
+  | "review";
+
+export const pageToUrl = (page: JobBuilderPage) => {
+  switch (page) {
+    case "intro":
+      return jobBuilderIntro;
+    case "details":
+      return jobBuilderDetails;
+    case "env":
+      return jobBuilderEnv;
+    case "impact":
+      return jobBuilderImpact;
+    case "tasks":
+      return jobBuilderTasks;
+    case "skills":
+      return jobBuilderSkills;
+    case "review":
+      return jobBuilderReview;
+  }
+};
```

Comment:
`pageToUrl` is missing a return type `((locale: string, jobId?: number | undefined) => string)`
Eslint is complaining that switch case doesn't have a `default`. Maybe you could set the default `jobBuilderReview`?